content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Data Services Cloud Console
+version: 0.1.1
+description: Easy to read guides, tips and documentation to rapidly help any
+  developer working with Data Services Cloud Console APIs.
+image: /img/platforms/image-1-dscc.svg
+width: large
+priority: 1
+tags:
+  - data-services-cloud-console
+  - greenlake-common-cloud
+---
+Today, every organization are required to unleash the power of data to drive digital transformation. But fragmented data management tools, manual processes, and infrastructure silos spanning edge to cloud are getting in the way. Customers across industries are struggling with the complexity of managing data and infrastructure, because it  creates a roadblock to innovation and agility, but also drives business risk. And it’s only getting harder as data continues to grow, apps continue to evolve, and infrastructure continues to spread edge-to-cloud.
```

Comment:
Recommend modifying the 2nd to last sentence and breaking it out into two: "Customers across industries... creates a roadblock to innovation and agility." Then, "It also amplifies business risk." Then continue with "And it's only getting harder..."

Reply:
Got it Dale, I break down the sentences to multiple ones and also use the conjunction to simplify them.

Reply:
Exactly. Beautiful!